### PR TITLE
DMP-796-Migration-DB

### PR DESCRIPTION
**Pull request by scott-robertson1**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by bancey — CHANGES_REQUESTED** [review]

File: database.tf [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  component           = var.component
+  business_area       = "sds"
+  location            = var.location
+  pgsql_storage_mb    = 4194304
+  pgsql_sku           = "GP_Standard_D8ds_v4"
+  
+
+  common_tags                        = var.common_tags
+  admin_user_object_id           = var.jenkins_AAD_objectId
+  pgsql_delegated_subnet_id = azurerm_subnet.postgres.id
```
[the remark this answers is not in the document]
Comment (scott-robertson1) [reply database.tf]: no joy, issue with the delegated subnets ?

File: database.tf [hunk 2]
```diff
@@ -0,0 +1,80 @@
+
+data "azurerm_resource_group" "rg" {
+  name = local.rg_name
+}
```
Comment (bancey) [inline database.tf:4]: Is this required? Doesn't look like its used
```suggestion
```

Comment (scott-robertson1) [reply database.tf:4]: is it not used in the subnet ?

Comment (bancey) [reply database.tf:4]: the `rg_name` local is being passed through to the `resource_group_name` parameter on the subnet. Can't see this in use there.

**Review by bancey — APPROVED** [review]

**Pull request merged**